### DISCUSSION TITLE
[Closed] Support arm64 on Linux

### DISCUSSION THREAD

**Discussion by ebroder** [meteor/meteor#12408] [Feature Requests]
With Meteor 2.5, the project added support for running on M1 Macs, but it would be great if it was possible to use Meteor on arm64-based Linux as well.

I have two main motivations for wanting this:

- I'd like to be able to run Meteor applications on AWS Graviton processors. They have generally been shown to be both cheaper and more performant than their Intel or AMD equivalents. (I'll note that GCP and Azure both also offer ARM instances, though I am not immediately aware of whether they have similar cost/performance improvements over Intel/AMD instances).
- I'd like to be able to debug my application's Docker build faster. When I test my application's Docker build locally in Docker for Mac, I have to run it with `--platform amd64`, since it's using a Linux container. This is significantly slower than running the application or its build process natively. (As far as I'm aware, Docker for Mac has to use qemu emulation rather than using Rosetta, although there is [an open issue](https://github.com/docker/roadmap/issues/384)) to change that).

(I'll note that this is related to #11661, although this is a more specific request, rather than expanding Meteor support to a broad range of architectures)

**Comment by rj-david** [comment]
https://github.com/meteor/meteor/pull/12160
